site/docs/tools/auth_config.md

```diff
@@ -29,6 +29,15 @@ or [Delta Lake](./deltalake/index.md)) the Nessie authentication settings are co
             .getOrCreate()
     ```
 
+## Hive
+
+When Nessie is used in Hive with [Iceberg](./iceberg/index.md), the Nessie authentication settings are configured through Hive Shell:
+
+```
+SET iceberg.catalog.<catalog_name>.catalog-impl=org.apache.iceberg.nessie.NessieCatalog
+SET iceberg.catalog.<catalog_name>.authentication.type=NONE
```

Comment:
Other sections in this doc use `nessie` as a specific example for `<catalog_name>` (e.g. the section just below). Would it make sense to use `nessie` here as well?

Alternatively, I guess we ought to use `<catalog_name>` in the other sections of this doc, which would also be fine. I think we have to use one way of the other consistently for the sake of clarity. WDYT?

Reply:
I think `<catalog_name>` makes more sense, more clear. Will change this 👍🏽

Reply:
Could you also update this file to use `<catalog_name>`, please?

Reply:
It is using `<catalog_name>` already, isn't? 🤔

Reply:
Only in this section. Other sections use literal "nessie" ATM

Reply:
Oh I have changed that already

Reply:
Did you push? I still see older text in GH :thinking:

Reply:
Sorry, apparently I didn't push it, it should be now better